src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/BlazorServerSide-CSharp.Client/Pages/FetchData.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@page "/fetchdata"
+@inject WeatherForecastService ForecastService
```

Comment:
Do you normally pascal case the thing being injected? Do you think of it as a property or a field? Can't remember which it actually is.

Reply:
Yes, it's a property